tools/pipelines/test-real-service.yml

```diff
@@ -101,44 +91,6 @@ stages:
         logger__colorize: "false" # Need to pass it as string so ADO doesn't convert it into False (capital F) which doesn't work
         logger__morganFormat: tiny
 
-  # end-to-end tests routerlicious
-  - template: /tools/pipelines/templates/include-test-real-service.yml@self
-    parameters:
-      stageId: e2e_routerlicious
-      stageDisplayName: e2e - routerlicious
-      poolBuild: Small-eastus2
-      testPackage: ${{ variables.testPackage }}
-      testWorkspace: ${{ variables.testWorkspace }}
-      artifactBuildId: $(resources.pipeline.client.runID)
-      testCommand: test:realsvc:routerlicious:report
-      lockBehavior: sequential
-      continueOnError: true
-      ${{ if eq(parameters.useR11sPpeEnvironment, true) }}:
-        r11sSelfSignedCertSecureFile: wu2-ppe-tls-certificate.pem
-      ${{ else }}:
-        r11sSelfSignedCertSecureFile: wu2-tls-certificate.pem
-      stageVariables:
-        - group: e2e-r11s-lock
-      splitTestVariants:
-        - name: Non-compat
-          flags: --compatVersion=0
-        - name: N-1
-          flags: --compatVersion=-1
-        - name: LTS
-          flags: --compatVersion=LTS
-        - name: Cross-Client
-          flags: --compatVersion=CROSS_CLIENT
-      cacheCompatVersionsInstalls: true
-      uploadTestPassRateTelemetry: true
-      pipelineIdentifierForTelemetry: ${{ variables.pipelineIdentifierForTelemetry }}
-      env:
-        ${{ if eq(parameters.useR11sPpeEnvironment, true) }}:
-          fluid__test__driver__r11s: $(automation-fluid-test-driver-r11s-ppe)
```

Comment:
Probably worth keeping track of the ADO variables that are losing their purpose so we can clean them up. The SSL certs also live in ADO secret files (IIR[the name]C) and an Azure key-vault.